app/templates/includes/proposition-links.html

```diff
@@ -1,6 +1,6 @@
-{% if session.passport.user %}
+{% if req.user %}
 <ul id="proposition-links">
-  <li><a href="{{ url_for('base.home') }}">Signed in as {{ session.passport.user.name }}</a></li>
+  <li><a href="{{ url_for('base.home') }}">Signed in as {{ user.name if user.name else user.email }}</a></li>
```

Comment:
I think these want to be `req.user.name` and `req.user.email`, because `user` may not always be passed to the template, but `req` is always available in templates (because [I added it to `app.locals`](https://github.com/ministryofjustice/analytics-platform-control-panel-frontend/blob/4f3a68fa1a24646587f7311c08ae86876aab410d/app/index.js#L55)).

Or, these could use `current_user` - which is also in `app.locals`. We could change that to `user` and then this code wouldn't have to change.

Reply:
Fine with changing this to `current_user` – if I'd thought to look in `app.locals` I would have used that anyway.